src/components/pages/collectRecordFormPages/CollectRecordFormPageAlternative/CollectRecordFormPageAlternative.js

```diff
@@ -241,44 +289,46 @@ const CollectRecordFormPageAlternative = ({
   ])
 
   useEffect(
-    function loadObservationsFromCollectRecordIntoState() {
-      if (!apiObservationsLoaded && collectRecordBeingEdited) {
-        const observationsFromApiTable1 =
-          collectRecordBeingEdited.data[
-            getObservationsPropertyNames(collectRecordBeingEdited)[0]
-          ] ?? []
-        const observationsFromApiTable2 =
-          collectRecordBeingEdited.data[
-            getObservationsPropertyNames(collectRecordBeingEdited)[1]
-          ] ?? []
-
-        const persistedUnsavedObservationsTable1 = getPersistedUnsavedObservationsTable1Data()
-        const persistedUnsavedObservationsTable2 = getPersistedUnsavedObservationsTable2Data()
-        const initialObservationsToLoadTable1 =
-          persistedUnsavedObservationsTable1 ?? observationsFromApiTable1
-        const initialObservationsToLoadTable2 =
-          persistedUnsavedObservationsTable2 ?? observationsFromApiTable2
-
-        observationsTable1Dispatch({
-          type: 'loadObservationsFromApi',
-          payload: initialObservationsToLoadTable1,
+    function initializeObservationReducerTable1() {
+      if (!isObservationsTable1ReducerInitialized) {
```

Comment:
added the `!isObservationsTable1ReducerInitialized` conditional here for easier code reasoning/readability.

Reply:
much easier to understand!